std/algebra/emulated/sw_bn254/pairing_test.go

```diff
@@ -71,7 +72,7 @@ func (c *FinalExponentiationCircuit) Define(api frontend.API) error {
 		return fmt.Errorf("new pairing: %w", err)
 	}
 	res := pairing.FinalExponentiation(&c.InGt)
-	pairing.ext12.assertIsEqual(res, &c.Res)
+	pairing.Ext12.AssertIsEqual(res, &c.Res)
```

Comment:
I previously added `AssertIsEqual` to the `Pairing` type, so it should be sufficient to do:
```go
pairing.AssertIsEqual(res, &c.Res)
```